docs/source/architecture/character.rst

```diff
@@ -0,0 +1,100 @@
+
+Alice may have data that she wants to share and it is conceivable that she would want to classify/categorize
+her data according to how she intends to share it. Each of these subsets should have their own set of permissions - within NuCypher
+this concept is called a `label`. Alice uses this `label`  to categorize data and she creates an asymmetric encryption key
+for Enrico characters based on the `label`. For example, Alice may use different `labels` and different Enricos for
+health data, personal data, and work-related data. Essentially, `labels` are a way to categorize data for sharing - you
+can think of file system folders as being somewhat analogous to `labels`. Alice can delegate access permissions to
+the encrypted data classified under a `label` to any recipient she sees fit.
+
+The same encryption key can be used by Enrico even though multiple policies for the data can be created
+for different Bobs. Enrico, therefore, only has to encrypt data using the provided encryption key
+and data can later be re-encrypted for all Bobs who have been granted access.
+
+When Alice wants to share this information with Bob, she can create a
+policy in the NuCypher Network to grant access to him. To grant access to Bob, Alice must:
+
+#. Create a re-encryption key based on the asymmetric private key and Bob's public key
+#. Configure the conditions of the policy e.g. *expiration time, m-of-n threshold values*
+#. Deploy the policy including associated fees to the blockchain
+#. Deploy `n` policy arrangements, each containing a re-encryption key fragment, `kFrag`, to `n` nodes on the NuCypher Network
+
+
+Enrico::Encrypt
+^^^^^^^^^^^^^^^
+
+.. image:: ../.static/img/enrico_encrypt.svg
+    :target: ../.static/img/enrico_encrypt.svg
+
+The NuCypher Network uses the Umbral_ threshold proxy re-encryption scheme which employs a key encapsulation mechanism (KEM)/data
+encapsulation mechanism (DEM). In this approach, an ephemeral symmetric key is used to encrypt the data, and the
+symmetric key is encrypted using an asymmetric encryption key. The encrypted data (the DEM portion) and the
+encrypted symmetric key (the KEM portion, called a `capsule`) are stored together.
+
+Even though different blocks of newly generated bulk data may use different symmetric keys, the same asymmetric
+encryption key provided by Alice is used to encrypt the symmetric key. How and when Enrico uses a new symmetric key
+is up to the user of the Enrico API.
+
+
+Bob::Retrieve
+^^^^^^^^^^^^^
+
+.. image:: ../.static/img/bob_retrieve.svg
+    :target: ../.static/img/bob_retrieve.svg
+
+When Bob wants to access the data, he must first obtain the encrypted data and `capsule` from encrypted storage.
+However, this data is currently encrypted and inaccessible to Bob.
+
+Bob will need to get the `capsule` re-encrypted from the encryption key configured by Alice to
+his public key. Remember that when Alice granted access to Bob, she distributed `n`
```

Comment:
> re-encrypted from the encryption key configured by Alice to his public key

I think the concept of a "configured encryption key" is misleading - This a "Re-Encryption Key", "CFrag" or "KFrag".

"...In order to gain access, Bob must request re-encryption of capsule fragments by n of the Ursulas Alice previously distributed kFrags to. "

Reply:
✔️